src/conventions/datapack_uninstallation.md

```diff
@@ -11,3 +11,105 @@ There are no rules enforcing how you should implement the uninstallation method,
 ### Uninstallation Function
 
 You can create an "uninstallation function", which when ran, will remove most of the content added from your datapack. You can tell the user about this function inside your project page or anywhere you like.
+
+--------------------
+
+### **Note from Convention Mod (Optional)**
+> I recommend if you have `scoreboard`, `storage`, `team`, you should remove it if possible.  
+> - If you can't remove something, such as `core scoreboard`, you can write the reason in the [request review](https://github.com/mc-datapacks/review-tracker/issues) or mcfunction comment.
+
+###  **What should you know and be aware of about how to disable datapack**
+**1. Use /datapack disable \<name>** - `tick.json`
+> Pros
+> - Using just 2 commands will clear the schedule command as well. 
+> - Auto reload, Help ensure that everything is completely cleared. Due to the use of the `/reload` command, some multiplayer servers have a server launcher. Changed this reload command so vanilla's commands it won't work.
+>
+> Cons
+> - Be careful \<name>
+>
+> Common problems
+> - In some cases I found that this command is not available in the uninstallation file, but even after removed `scoreboard`, `storage`, `team` the tick.json file will still run causing the uninstallation to fail.
+> - Another case to be aware of is \<name> . Because if you use `datapack disable "file/convention"`
+If the user installs a `.zip`, it won't work. And the file name must match as well.
+Even if you can do like this
+> ```mcfunction
+> datapack disable "file/convention"
+> datapack disable "file/convention.zip"
+>```
+> But if file name is `convention_v1.0.0.zip`.
+Either install it as a `.zip` or `extract to folder` it won't work.
+>```
+> <name>.zip -> /data
+> <name>/data
+>```
+
+**2. Use /schedule clear \<function>** - `load.json only`
+### **How to use**
+Typically you will use `load.json` to call mcfunctions like load, init, etc. to install scoreboard objectives, etc. as you need. which these files will only run once
+
+But if you have to use the same functionality as `tick.json`, you simply use `load.json` to call your main or tick function and use the `schedule function convention:datapack/main 1t` on the last line.
+
+**Fun Fact :** If you are a beginner, you may not know that you can make mcfunction run slower tick cycles as you like.
+For example, run every 8 tick `schedule function <function> 8t` or every 1 second `schedule function <function> 20t or 1s` `t - tick`, `s - second`, `d - gameday`. Why am I talking about this? Because I've seen scoreboards used to do this before.

```

Comment:
I brought it here because when giving advice on uninstalling, I need to recommend this as well. Because it is related
Either using `schedule 1t` instead of `tick.json`, uninstalling the datapack without using the `/datapack disable` command.